examples/CMakeLists.txt

```diff
@@ -84,11 +106,16 @@ foreach(EXAMPLE_DIR ${EXAMPLE_DIRS})
         # Setup the runtime dependencies of Vex
         vex_setup_runtime(${EXAMPLE_NAME})
 
+        if(EXAMPLE_NAME STREQUAL "pix_events")
```

Comment:
This could probably be more consistent, aka either including pix and renderdoc everywhere, or including them only in their specific exampels.